producers/mysql-producer/src/main/java/com/flipkart/aesop/runtime/producer/eventhelper/BinLogEventHelper.java

```diff
@@ -0,0 +1,20 @@
+package com.flipkart.aesop.runtime.producer.eventhelper;
+
+import com.google.code.or.common.glossary.Column;
+import com.google.code.or.common.glossary.Row;
+
+import java.util.List;
+
+/**
+ * Created by akshit.agarwal on 11/03/16.
+ */
+public class BinLogEventHelper
+{
+    public static void appendColumnToRow(Row actualRow, Column columnToAppend)
+    {
+        List<Column> oldColValues = actualRow.getColumns();
+        oldColValues.add(columnToAppend);
+        actualRow.setColumns(oldColValues);
```

Comment:
Is this necessary? If the reference to the list returned is maintained, then you don't need to set it again. Row seems to be just a POJO (http://grepcode.com/file/repo1.maven.org/maven2/com.zendesk/open-replicator/1.2.0/com/google/code/or/common/glossary/Row.java)